src/main/controllers/AboutHearingDocumentsController.ts

```diff
@@ -0,0 +1,164 @@
+import { Response } from 'express';
+
+import { Form } from '../components/form/form';
+import { isFieldFilledIn } from '../components/form/validator';
+import { AppRequest } from '../definitions/appRequest';
+import { TranslationKeys } from '../definitions/constants';
+import { FormContent, FormFields } from '../definitions/form';
+import { AnyRecord } from '../definitions/util-types';
+import { getLogger } from '../logger';
+
+import { assignFormData, getPageContent } from './helpers/FormHelpers';
+import { getLanguageParam } from './helpers/RouterHelpers';
+
+const logger = getLogger('AboutHearingDocumentsController');
+
+export default class AboutHearingDocumentsController {
+  private form: Form;
+  private aboutHearingDocumentsContent: FormContent;
+
+  constructor() {
+    // intentionally empty
+  }
+
+  public post = async (req: AppRequest, res: Response): Promise<void> => {
+    const { userCase } = req.session;
+    req.session.errors = [];
+    const errorsForPage = () => {
```

Comment:
Please move this out of the controller into a helper class if possible

Reply:
this has been moved.